Papers/01_cell_values_derivation.Rmd

```diff
@@ -228,7 +228,7 @@ lyt <- basic_table() %>%
    analyze(c("SEX", "AGE"), afun = function(x) {
       if (is.numeric(x)) {
          in_rows(
-            "n" = sum(is.na(x)),
+            "Missing" = sum(is.na(x)),
```

Comment:
Thank you for noticing this.  You're correct there is a bug here, but it is in the other direction. Can you please change the PR to make that `sum(!is.na(x))` with the name remaining `"n"`?